default/templates/default/base-tool.html

```diff
@@ -38,12 +38,10 @@
     <span class="file open-drive-link hidden">
         {% trans "File saved on Google Drive" %}&nbsp;<a target="_blank" role="button"> {% trans "Open" %}</a>
     </span>
-    <div class="form-group form-inline form-send hidden">
+    <div class="form-group form-inline form-send">
```

Comment:
I think you could include the `<div>` inside the tag definition, so you won't need to use javascript to hide it when empty.

Reply:
Done.